.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,79 @@
+# .github/workflows/goreleaser.yml
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
+
+permissions:
+  contents: write
```

Comment:
I just want to double-check, why do we require write permissions?

Reply:
From their docs: https://goreleaser.com/ci/actions/#token-permissions

> - contents: write if you wish to
>   - [upload archives as GitHub Releases](https://goreleaser.com/customization/release/), or
>   - publish to [Homebrew](https://goreleaser.com/customization/homebrew/), or [Scoop](https://goreleaser.com/customization/scoop/) (assuming it's part of the same repository)
> - or just contents: read if you don't need any of the above
> - packages: write if you [push Docker images](https://goreleaser.com/customization/docker/) to GitHub
> - issues: write if you use [milestone closing capability](https://goreleaser.com/customization/milestone/)

Probably a good point to discuss if we need to attach the resulting binaries to the Github release 🤔

Reply:
I think it is fine 👍. I just want to double-check. Pipeline just runs on new tags.